File: Project/Routine.java
import java.util.ArrayList;
import java.util.List;

public class Routine {
    private int id;
    private List<Duty> duties;

    public Routine() {
        duties = new ArrayList<>();
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public List<Duty> getDuties() {
        return duties;
    }

    public void addDuty(Duty duty) {
        duties.add(duty);
    }

    public void removeDuty(Duty duty) {
        duties.remove(duty);
    }
}

File: StaffsManagementSys_winform/StaffsManagementSys_winform/Models/Routine.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace StaffsManagementSys_winform.Models
{
    public class Routine
    {
        public int Id { get; set; }
        public List<Duty> Duties { get; set; }

        public Routine()
        {
            Duties = new List<Duty>();
        }

        public void AddDuty(Duty duty)
        {
            Duties.Add(duty);
        }

        public void RemoveDuty(Duty duty)
        {
            Duties.Remove(duty);
        }

        public string GetRoutineSummary()
        {
            return $"Routine ID: {Id}, Total Duties: {Duties.Count}";
        }
    }
}
